nanoFramework.Device.Bluetooth/GenericAttributeProfile/GattSession.cs

```diff
@@ -4,23 +4,34 @@
 //
 
 using System;
+using System.Diagnostics;
 
 namespace nanoFramework.Device.Bluetooth.GenericAttributeProfile
 {
     /// <summary>
     /// This class represents a GATT session.
     /// </summary>
-    public class GattSession : IDisposable
+    public class GattSession
     {
         private readonly BluetoothDeviceId _deviceId;
+        private ushort _maxPduSize;
 
         /// <summary>
-        /// Dispose GattSession object
+        /// Delegate for SessionStatusChanged events.
         /// </summary>
-        public void Dispose()
-        {
-            // not used yet
-        }
+        /// <param name="sender">GattSession sending event</param>
+        /// <param name="args">Event arguments</param>
+        public delegate void GattSessionStatusChangedEventHandler(Object sender, GattSessionStatusChangedEventArgs args);
+
+        /// <summary>
+        /// Session status change event.
+        /// </summary>
+        public event GattSessionStatusChangedEventHandler SessionStatusChanged;
+
+        /// <summary>
+        /// Maximum MTU size changed event
```

Comment:
What is MTU? Possible to put the full name on top of the abbreviation?
```suggestion
        /// Maximum MTU size changed event.
```